strapi/strapi
# feat(content-releases): Releases list page UI implementation

### What does it do?

Adds the UI to the Releases view page. Shows all the releases in a card, the number of releases

### Why is it needed?

To add real content to the releases page

### How to test it?

You can go to the Release page and create new Releases
- at the beginning you have 0 releases in the Releases view page and the empty content section is shown
- when you add new releases you can see the subtitle of the page reflecting the number of releases and a new card in the Pending tab is shown
- inside the card you can see the release name, you can click in every single card to open the Release Details page of the release
- there is also Pagination, default value 16
